projects/1.20/assets/project-expansion/projectexpansion/lang/zh_cn.json

```
@@ -0,0 +1,582 @@
+	"advancements.projectexpansion.collector": "更强的能量收集器",
+	"advancements.projectexpansion.collector.description": "现在荒谬的事情开始了。",
+	"advancements.projectexpansion.compressed_collector": "嘿，看我把收集器压缩成了什么！",
+	"advancements.projectexpansion.compressed_collector.description": "现在它们没用了，直到它们被制作成别的东西。",
+	"advancements.projectexpansion.final_collector": "终极收集器",
+	"advancements.projectexpansion.final_collector.description": "你需要这么多EMC吗？",
+	"advancements.projectexpansion.final_compressed_collector": "终极压缩收集器",
+	"advancements.projectexpansion.final_compressed_collector.description": "宇宙可能就要大爆炸了。",
+	"advancements.projectexpansion.relay": "迈向真正力量花盆栽罐的第一步。",
+	"advancements.projectexpansion.relay.description": "要到达巅峰还有很长的路。",
+	"advancements.projectexpansion.final_relay": "终极中继器",
+	"advancements.projectexpansion.final_relay.description": "说真的，你需要这么多EMC吗？",
+	"advancements.projectexpansion.power_flower": "真正的力量花盆",
+	"advancements.projectexpansion.power_flower.description": "现在你在用煤气做饭。",
+	"advancements.projectexpansion.final_power_flower": "终极力量花盆",
+	"advancements.projectexpansion.final_power_flower.description": "请去摸摸一些草。",
+	"advancements.projectexpansion.EMC_link": "不是，你真能做到这一步啊？",
+	"advancements.projectexpansion.EMC_link.description": "能量凝聚器什么的都去死吧。",
+	"advancements.projectexpansion.final_emc_link": "终极EMC接口",
+	"advancements.projectexpansion.final_emc_link.description": "去给我的EMC浇点水吧。",
+	"advancements.projectexpansion.final_star": "终极之星",
+	"advancements.projectexpansion.final_star.description": "大功告成，你可以出去走走了。",
+	"advancements.projectexpansion.basic_alchemical_book": "基础炼金之书",
+	"advancements.projectexpansion.basic_alchemical_book.description": "获得一本基础炼金之书。",
+	"advancements.projectexpansion.advanced_alchemical_book": "高级炼金之书",
+	"advancements.projectexpansion.advanced_alchemical_book.description": "获得一本高级炼金之书。",
+	"advancements.projectexpansion.master_alchemical_book": "大师炼金之书",
+	"advancements.projectexpansion.master_alchemical_book.description": "获得一本大师炼金之书。",
+	"advancements.projectexpansion.arcane_alchemical_book": "奥术炼金之书",
+	"advancements.projectexpansion.arcane_alchemical_book.description": "获得一本奥术炼金之书。",
+	"advancements.projectexpansion.knowledge_sharing_book": "分享就是关爱！",
+	"advancements.projectexpansion.knowledge_sharing_book.description": "但我并不在乎。",
+	"advancements.projectexpansion.compact_sun": "汝掌心中者，太阳之力也。",
+	"advancements.projectexpansion.compact_sun.description": "不要盯着它太久哦……",
+	"advancements.projectexpansion.blinded_by_the_light": "被光芒蒙蔽",
+	"advancements.projectexpansion.blinded_by_the_light.description": "你被警告了。",
+	"block.projectexpansion.advanced_alchemical_chest.color_set": "颜色已设置为%s。",
+	"block.projectexpansion.advanced_alchemical_chest.invalid_item": "你必须手持一个%s来设置颜色。",
+	"block.projectexpansion.advanced_alchemical_chest.color": "当前颜色：%s",
+	"block.projectexpansion.advanced_alchemical_chest.tooltip": "就像炼金术之袋一样，但以箱子的形式呈现。按住shfit键点击炼金术之袋来设置颜色。",
```

```suggestion
	"block.projectexpansion.advanced_alchemical_chest.tooltip": "和炼金术之袋类似的箱子。手持炼金术之袋，按住Shift并点击来设置颜色。",
```
前半句减少生硬翻译，后半句语义整个就错了